data/repos.yml

```diff
@@ -539,6 +674,17 @@
   sentry_url: false
   dashboard_url: false
 
+# Assume this should be added to Data Products' portfolio, based on `govuk-knowledge-graph`
+- repo_name: govuk-knowledge-graph-gcp
+  private_repo: true # well not really, but it's "internal"
```

Comment:
No longer internal - it's public.

Also should probably be added, as per https://github.com/alphagov/govuk-developer-docs/pull/3913/files#r1132281588

Reply:
Although given its experimental nature, perhaps it's not that important for GOV.UK to be aware of it.

@nacnudus do you have thoughts on this?